components/nimbus/android/src/main/java/org/mozilla/experiments/nimbus/Nimbus.kt

```diff
@@ -285,6 +318,7 @@ data class NimbusDeviceInfo(
 class NimbusDelegate(
     val dbScope: CoroutineScope,
     val fetchScope: CoroutineScope,
+    val updateScope: CoroutineScope? = null,
```

Comment:
nit: would be nice to have a KDoc on this so we know what it's use case within Nimbus is for in the future. To avoid scope creeping this comment, we can add a doc like only for this param for now; more docs can be added in a follow-up in a future change.